manifests/t/Trillian/Trillian/6.5.0.24/Trillian.Trillian.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Trillian.Trillian
+PackageVersion: 6.5.0.24
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://trillian.im/get/windows/6.5/trillian-v6.5.0.24.msi
+  InstallerSha256: 6DE1F9E452FEE3AB56138AF2D130CBC5574F5BD5CC9899B0A384F5A0945F12BA
+  ProductCode: '{E614066A-AC6F-4CA1-84CB-266A51F7FC42}'

```

```suggestion
  ProductCode: '{E614066A-AC6F-4CA1-84CB-266A51F7FC42}'
  AppsAndFeaturesEntries:
  - DisplayName: Trillian Machine-Wide Installer
    DisplayVersion: 6.5.24
    ProductCode: '{E614066A-AC6F-4CA1-84CB-266A51F7FC42}'
```
This should solve https://github.com/microsoft/winget-pkgs/labels/Manifest-AppsAndFeaturesVersion-Error